### ISSUE TITLE
build dashboard-server-frontend in docker

### ISSUE THREAD

**Issue by Gallardot** [pingcap/chaos-mesh#376]
## Feature Request

**Is your feature request related to a problem? Please describe:**

I want to build docker images
```
make image
```
I get some error logs
```
Step 5/6 : COPY build /web
COPY failed: stat /var/lib/docker/tmp/docker-builder449059249/build: no such file or directory
```

 https://github.com/pingcap/chaos-mesh/blob/30567158b57559e4c8e34e76da8e7483089bf491/images/chaos-dashboard/Dockerfile#L8

Huh?  It looks like we need to build the front end first.
```
make dashboard-server-frontend
```

I also get some error logs
```
cd images/chaos-dashboard; yarn install; yarn build
bash: yarn: command not found
bash: yarn: command not found
make: *** [dashboard-server-frontend] Error 127
```

We have to install yarn!

**Describe the feature you'd like:**

I think chaos-mesh is a Back-end project. Install yarn maybe not friendly for non-front-end developers.

**Describe alternatives you've considered:**

Base on the [chaos-dashboard/Dockerfile](https://github.com/pingcap/chaos-mesh/blob/30567158b57559e4c8e34e76da8e7483089bf491/images/chaos-dashboard/Dockerfile)，build the dashboard-server-frontend on a nodejs images，and copy the result to the  chaos-dashboard images.

**Teachability, Documentation, Adoption, Migration Strategy:**

**Comment by Gallardot** [comment]
I made an attempt.  #377

**Comment by cwen0** [comment]
@Gallardot Can we close this issue?

**Comment by Gallardot** [comment]
fix in #377
